LiskHQ/lisk-hub
# Add create Twitter Newsfeed component - Closes #913

### What was the problem?
No Newsfeed component
### How did I fix it?
Implemented it
### How to test it?
- Go to dashboard
- You should see 
https://user-images.githubusercontent.com/1254342/41159896-58d2b99c-6b2e-11e8-9c13-fb4d9f1135bb.png
with login and without.
- Read me should open new tab with source of feed
- In Right corner there is `pencil icon` where you can select which feed to show
(Currentely there is only Twitter feed)

### Review checklist
- The PR solves #913 
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
